core/web/webauthn_controller.go

```diff
@@ -20,12 +19,12 @@ import (
 // with those keys
 type WebAuthnController struct {
 	App                          chainlink.Application
-	InProgressRegistrationsStore *sessions.WebAuthnSessionStore
+	inProgressRegistrationsStore *sessions.WebAuthnSessionStore
 }
 
 func (c *WebAuthnController) BeginRegistration(ctx *gin.Context) {
-	if c.InProgressRegistrationsStore == nil {
-		c.InProgressRegistrationsStore = sessions.NewWebAuthnSessionStore()
+	if c.inProgressRegistrationsStore == nil {
+		c.inProgressRegistrationsStore = sessions.NewWebAuthnSessionStore()
```

Comment:
This looks like a race, no?

Reply:
Internals too I think. I will follow up: https://app.shortcut.com/chainlinklabs/story/20712/make-webauthnsessionstore-concurrent-safe